android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -768,6 +769,11 @@ protected void onSaveInstanceState(Bundle outState)
         fragment.saveAltitudeChartState(outState);
     }
 
+    if(mNavigationController != null)
```

Comment:
Please download the out code style scheme https://www.dropbox.com/s/b6xbx5ajryh7rhx/maps.me.xml?dl=0 and apply it to your Android Studio. You just need to place this xml file into this directory /Users/<your_user>/Library/Preferences/AndroidStudio2.2/codestyles/maps.me.xml (if your OS is MacOs, if not - please google it) . After that you should restart your Android studio and choose this scheme here http://clip2net.com/s/3CLs8ZA 

Of course, it's not a requirement and you can fix formatting manually, but probably you find it convenient.

P.S: white space are missed. And we don't put brackets with if condition when the body is just single line (take a look at the same place in our code please)